experimental/generated/beats/fields.ecs.yml

```diff
@@ -969,6 +999,16 @@
       description: Subject name of the code signer
       example: Microsoft Corporation
       default_field: false
+    - name: code_signature.team_id
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: 'The team identifier used to sign the process.
```

Comment:
I think for both of these, it's worth noting that it's mostly relevant to macOS or other Apple operating systems. It might not be intuitive for people trying to map this field to a Windows sensor, like Elastic Endpoint or Sysmon.